data/model/oci/tag.py

```diff
@@ -184,6 +185,13 @@ def list_repository_tag_history(
         .offset(page_size * (page - 1))
     )
 
+    if filter_tag_name is not None:
+        operation, value = filter_tag_name.split(':')
```

Comment:
will this have a runtime error if the filter does not contain a ":" ?

Reply:
Good catch. Added try except to return 400: Bad response when syntax doesn't match.